runtime/compiler/control/CompilationThread.cpp

```diff
@@ -6267,9 +6405,73 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
             *aotCachedMethod = javaVM->sharedClassConfig->findCompiledMethodEx1(vmThread, J9_ROM_METHOD_FROM_RAM_METHOD(method), &flags);
             if (!(flags & J9SHR_AOT_METHOD_FLAG_INVALIDATED))
                {
+#ifdef COMPRESS_AOT_DATA
+               TR_AOTMethodHeader *aotMethodHeader = (TR_AOTMethodHeader*)(((J9JITDataCacheHeader*)*aotCachedMethod) + 1);
+               if (aotMethodHeader->flags & TR_AOTMethodHeader_CompressedMethodInCache)
+                  {
+                  /**
+                    * For each AOT compiled method we store in the shareclass cache we have follwing layout.
```

Comment:
typo: follwing --> following

Reply:
Fixed in https://github.com/eclipse/openj9/pull/5926/commits/66c4368333f9f5ba388b64ec16893d59d9c7eaa1